package.json

```diff
@@ -137,6 +137,50 @@
           ],
           "default": "on",
           "description": "Refresh memory views when debugger stops"
+        },
+        "memory-inspector.wordsPerGroup": {
+          "type": "number",
+          "enum": [
+            1,
+            2,
+            4,
+            8,
+            16
+          ],
+          "default": 1,
+          "description": "Default words per group"
+        },
+        "memory-inspector.groupsPerRow": {
+          "type": "number",
+          "enum": [
+            1,
+            2,
+            4,
+            8,
+            16,
+            32
+          ],
+          "default": 4,
+          "description": "Default groups per row"
+        },
+        "memory-inspector.variablesVisible": {
+          "type": "boolean",
+          "default": false,
+          "description": "Show variables column?"
+        },
+        "memory-inspector.asciiVisible": {
+          "type": "boolean",
+          "default": false,
+          "description": "Show ASCII column?"
+        },
+        "memory-inspector.scrollingBehavior": {
+          "type": "string",
+          "enum": [
+            "Paginate",
+            "Infinite"
+          ],
+          "default": "Paginate",
+          "description": "Behavior when adding more memory beyond the current view. 'Infinite' appends new memory to bounds of current request, resulting in a growing list. 'Paginate' maintains a consistent memory size, replacing the previous request."
```

Comment:
You can use an `enumDescriptions` field to match enum options to a description of their behavior, rather than including it in the description of the preference.

Reply:
Nice, I've added enum descriptions in place of this lengthy description